cfme/containers/image_registry.py

```diff
@@ -40,3 +46,17 @@ def navigate(self, detail=True):
         else:
             sel.force_navigate('containers_image_registry',
                 context={'image_registry': self})
+
+    def get_detail(self, *ident):
```

Comment:
This is obsolete. Use `provider.summary.something`, not `provider.get_detail()`.